app/v3/models/requestData/ListBFLossesRawData.scala

```diff
@@ -16,5 +16,7 @@
 
 package v3.models.requestData
 
-case class ListBFLossesRawData(nino: String, taxYear: Option[String],
-                               typeOfLoss:Option[String], businessId: Option[String]) extends RawData
+case class ListBFLossesRawData(nino: String,
+                               taxYear: Option[String],
```

Comment:
taxYear -> taxYearBroughtForwardFrom
our models should match our requirements spec